### DISCUSSION TITLE
How to add more coverage metrics in pull request comment, such as function and statement coverages?

### DISCUSSION THREAD

**Discussion by keita-determined** [codecov/feedback#281] [Q&A] [answered]
Codecov UI shows `Statement` and `Function` coverage like the following screenshot.

<img width="1706" alt="image" src="https://github.com/codecov/feedback/assets/109113616/cae22ccf-ded7-4073-afaa-6ae63a2c1c30">

However, these coverage metrics are not visible in the pull request comment.

```diff
@@            Coverage Diff             @@
##             master    #8709      +/-   ##
==========================================
- Coverage   47.58%   42.67%   -4.91%     
==========================================
  Files        1068      751     -317     
  Lines      170311   131436   -38875     
  Branches     2243     2244       +1     
==========================================
- Hits        81037    56096   -24941     
+ Misses      89116    75182   -13934     
  Partials      158      158              
```

How can I add these metrics into the PR comment?

**Comment by rohan-at-sentry** [comment, marked as answer]
Hi @keita-determined  - this is currently not supported. I'm curious : what would be the benefit for you/your team if this was added in?

**Reply by keita-determined** [reply]
i see. thanks!
It's because there are  many imported/exported functions, so it's import to know which function is tested or not.
